recipes/_master_base_config.rb

```diff
@@ -28,6 +28,38 @@
 # Get VPC CIDR
 node.default['cfncluster']['ec2-metadata']['vpc-ipv4-cidr-block'] = get_vpc_ipv4_cidr_block(node['macaddress'])
 
+# Get shared_dir path and mount EFS filesystem
+efs_shared_dir = "/" + node['cfncluster']['cfn_efs_shared_dir'].split(',')[0]
+
+# Check to see if EFS is created
+if efs_shared_dir != "/NONE"
+  # Create the EFS shared directory
+  directory efs_shared_dir do
+    owner 'root'
+    group 'root'
+    mode '1777'
+    recursive true
```

Comment:
why recursive on create?

Reply:
so recursive will ensure all the root directory have the same owner group mode permissions. so if the user wants the efs directory to be "/dirs/efs" and recursive is true, both "/dirs" and "/dirs/efs" will have the owner group mode specified. If this is not a behavior that we want, we can take recursive out and that way only "/dirs/efs" will have the owner group mode specified.

Think about this now, I guess if we want to keep recursive in this block we should also set recursive to true in the directory block after the nfs mounting